src/components/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Button.css';
+
+function Button({ children, clickHandler, disabled }) {
+  return (
+    <button
+      className='button'
+      disabled={disabled === null}
```

Comment:
Waarom een null check terwijl het hier om een boolean gaat? Als er geen boolean in dit component komt, dan gaat het bij het gebruik van het Button component al mis. Als het goed is kun je hier dan gewoon `disabled={disabled}` doen